diff --git "a/tables.sql" "b/tables.sql"
index b141835..0fa6c9b 100644
--- "a/tables.sql"
+++ "b/tables.sql"
@@ -33,8 +33,8 @@ CREATE TABLE `users` (
   `files_in_used` bigint(20) unsigned NOT NULL default '0',
   `files_out_used` bigint(20) unsigned NOT NULL default '0',
   `login_count` int(11) unsigned NOT NULL default '0',
-  `last_login` datetime NOT NULL default '0000-00-00 00:00:00',
-  `last_modified` datetime NOT NULL default '0000-00-00 00:00:00',
+  `last_login` datetime NOT NULL default '1000-01-01 00:00:00',
+  `last_modified` datetime NOT NULL default '1000-01-01 00:00:00',
   PRIMARY KEY  (`id`),
   UNIQUE KEY `userid` (`userid`)
 ) ENGINE=MyISAM DEFAULT CHARSET=latin1 COLLATE=latin1_general_ci COMMENT='ProFTPd user table';
